packages/compartment-mapper/src/node-powers.js

```diff
@@ -40,18 +40,43 @@ const makeReadPowersSloppy = ({ fs, url = undefined, crypto = undefined }) => {
   const pathToFileURL =
     url === undefined ? fakePathToFileURL : url.pathToFileURL;
 
+  let readMutex = Promise.resolve(undefined);
+
   /**
    * @param {string} location
    */
   const read = async location => {
+    const promise = readMutex;
+    let release = Function.prototype;
```

Comment:
Weird.  I had to stare at this to see the value wasn't used for anything except typing.

Reply:
Indeed and it avoids an allocation of an otherwise pointless noop function, or a pointless runtime assertion. But, there isn’t a winning move here, as far as I can tell.